composer.json

```diff
@@ -27,7 +27,10 @@
         "drupal/helfi_tunnistamo": "^2.0",
         "drupal/redis": "^1.5",
         "drupal/stage_file_proxy": "^1.2",
-        "drush/drush": "^10.4"
+        "drush/drush": "^10.4",
+        "drupal/devel": "^4.1",
+        "drupal/restui": "^1.20",
+        "drupal/file_entity": "^2.0@beta"
```

Comment:
File entity module has been deprecated for a while and should not be used anymore. Quote from the module page (https://www.drupal.org/project/file_entity):

> D8 (do not use the D8 file_entity module unless you already had it installed prior to 8.4.x or unless you really want to and have good reason to be doing so), there are available modules (see links below) to help you migrate data to the D8 core media module. D8 Core 8.4.x and higher a module called media (not to be confused with the contrib media project of the same name). If you are using D8 file_entity you should use the core media module instead of file_entity. If you were previously using file_entity in D8 [D8 file_entity have a look at this link.](https://www.drupal.org/project/migrate_file_to_media) and the documentation[ more D8 documentation](https://www.liip.ch/de/blog/migrate-file-entities-to-media-entities-in-drupal-8). Appologies for the confusion for some D8 people that installed file_entity instead of using the core media module, this is because the core media module was introduced in 8.4.x and not 8.0.x. We will continue to support existing installs of the D8 version of file_entity but urge new installs to be avoided.

Basically, file_entity entities should be replaced with core's media entities.

Reply:
I took it out